README.md

```diff
@@ -48,15 +81,35 @@ How do folks in the community use RFC 4122 UUIDs?
 
 ## FAQ
 
-## History
+**what are the advantages to uuid being a built-in module?**
+
+- The `uuid` module is relied on by `> 2,600,000` repos on GitHub (June 2019). Guaranteeing a
+  secure, consistent, well-maintained `uuid` module provides value to millions of developers.
+- The 12 kb `uuid` module is downloaded from npm `> 62,000,000` times a month (June 2019); making
+  it a built-in module eventually saves TBs of bandwidth globally. If we continue to address user
+  needs, such as `uuid`, as we expand built-in modules, bandwidth savings add up.
 
 ## Specification
 
-## Implementations
+_to come..._
```

Comment:
Nit: You can probably leave specification as a TODO list item for now, rather than a section.